src/Web/E2E/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "dotnet-podcasts",
+  "version": "1.0.0",
+  "description": "---\r page_type: sample\r description: \".NET 6 reference application shown at .NET Conf 2021 featuring ASP.NET Core, Blazor, .NET MAUI, Microservices, and more!\"\r languages:\r - csharp\r products:\r - dotnet-core\r - ef-core\r - blazor\r - dotnet-maui\r - azure-sql-database\r - azure-storage\r - azure-container-apps\r - azure-container-registry\r - azure-app-service-web\r ---",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MarcusFelling/dotnet-podcasts.git"
```

Comment:
I am not sure how this works, but should the url be pointing to the main repo instead of the fork?

Reply:
Nice catch! Yes, it should be pointing to main repo.

Reply:
Fixed